otri/filtering/filters/interpolation_filter.py

```diff
@@ -14,56 +14,75 @@
 class InterpolationFilter(Filter):
     '''
     Interpolates value between two stream atoms if their time difference is greater than a given maximum interval.
-    The resulting atoms will have given values interpolated
+    The resulting atoms will have given values interpolated.
     Input:
         Oredered by datetime atoms.
     Output:
         Atoms interpolated for the given target interval
     '''
 
-    def __init__(self, input_stream: Stream, keys_to_change: Collection[str], target_interval: str = "minutes"):
+    def __init__(self, input: str, output: str, keys_to_interp: Collection[str], target_interval: str = "minutes"):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'input'](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5649486)